solutions/swb-reference/src/templates/README.md

```diff
@@ -1,30 +1,48 @@
+
+----
+## Integrating with a Central Network Account
+The following CFN templates help setting up hosting accounts while integrating them with a Central Network Account, using Transit Gateway and Network Firewall.
+## Firewall and TGW
+A Central Network account could be available with some customers who have specific network monitoring and regulatory requirements to be carried out with that account, thereby allowing the user manage main account and hosting account network traffic all at one place. 
+
+This [template](./new-firewall-vpc.yaml) can be deployed in such a central network account for that purpose. This uses a network firewall AMI and creates a Security VPC, Gateway Load Balancer with firewall instance targets, and a Transit Gateway. The Security VPC can be used for monitoring network traffic centrally between main account and hosting account(s).
+
+This template was originally designed for use with Palo Alto Networks' VM Series firewall offering on AWS Marketplace. However, the template can be modified to leverage y firewall offering on AWS Marektplace that has an AMI.
+
+## Onboard Account TGW
+This [template](./onboard-account-tgw.cfn.yaml) onboards a hosting account as well as helps integrate with the resources created as part of the `new-firewall-vpc.yaml` template deployed in the Central Network account. You need to provide the Gateway Load Balancer Service ID and Transit Gateway ID output values from the Central Network account to this template's input parameters. This template provisions network components (VPC and Subnets) based on the public/private nature selected during stack creation.
 
 ### Dependencies
-As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs. This may include removing the EC2 firewall and the Gatewayload balancer, and updating the Transit Gateway's and onboarding account's route tables to reroute trafic that used to go through the EC2 firewall.
+As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs.
 
 ### How to use these:
-1. Launch the stack in a network account
+1. Launch the stack in a central network account
 1. Share the network account's transit gateway with the hosting account
-    1.Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
+    1. Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
     1. Select the transit gateway and, from the Actions menu, select "Share transit gateway" 
```

Comment:
I know you didn't write this part, but I'd split this up into two separate actions: selecting the gateway and selecting "share"

Reply:
Keeping as-is for now.